apps/web/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "web",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@fluentui-react-native/checkbox": "0.3.50",
+    "@testing-library/jest-dom": "^4.2.4",
```

Comment:
What happens if you get rid of the @testing-library dependencies? They may end up being unneccessary. Lets opt for a minimal set of dependencies.